changelog/undistributed/changelog_show_security_policies_hit-count-23aug2021.rst

```diff
@@ -0,0 +1,6 @@
+--------------------------------------------------------------------------------
+                            New
+--------------------------------------------------------------------------------
+* JUNOS
+    * Added howSecurityPoliciesHitCount:
```

Comment:
I misscopied earlier and cut off the S, my apologies. Please add the capitol S back to the start of ShowSecurity

Reply:
Fixed and pushed.